source/Cosmos.System2/Graphics/Fonts/BitFont.cs

```diff
@@ -0,0 +1,143 @@
+using Cosmos.System.Graphics;
+using System.Collections.Generic;
+using System.Drawing;
+using System.IO;
+
+namespace Cosmos.System.Graphics.Fonts
+{
+    public struct BitFontDescriptor
+    {
+        public string Charset;
+        public MemoryStream MS;
+        public int Size;
+        public BitFontDescriptor(string aCharset, MemoryStream aMS, int aSize)
+        {
+            this.Charset = aCharset;
+            this.MS = aMS;
+            this.Size = aSize;
+        }
+    }
+
+    /// <summary>
+    /// For More https://github.com/nifanfa/BitFont
```

Comment:
Please don't redirect users to another repo.
 If you want to add further functionality into Cosmos, please update/create an article in the Docs/ subdirectory.

Reply:
sure. i just forgot to remove it.